### PR TITLE
Upload tinitex artifacts

### PR DESCRIPTION
**Pull request by yihui**
so that we don't need to rebuild them in the rstudio/tinytex repo.

### CODE REVIEW

**Comment by cderv** [comment]
That is an excellent solution !  thanks ! 

We could then just download them and surface them in `tinytex-releases` Github release page. 

Would it make sense otherwise to add the `tinitex` binaries to each pre-release and release here on Github ? Probably better in their own  release at `tinytex-releases` repo though.

**Comment by yihui** [comment]
I still consider `tinitex` experimental (my main concern is that the file size is too big), so I hope to "release" it in stealth.

I'm trying to build and upload it in this repo because I finally figured out why the build was failing in `rstudio/tinytex` in the past few days. It was due to the `setup-r` action, which might have changed `PATH`, causing a wrong `find` being used. I'm not sure what the exact problem is, but building `tinitex` *before* installing R worked okay.

Anyway, I don't have write access here. If you do, please feel free to merge it. Thanks!

**Review by cderv — APPROVED** [review]
> It was due to the setup-r action, which might have changed PATH, causing a wrong find being used. I'm not sure what the exact problem is, but building tinitex before installing R worked okay.

Interesting 🤔  nothing come to mind of what could have changed. 

> Anyway, I don't have write access here. If you do, please feel free to merge it. Thanks!

I have but usually only do PR and rely on @dragonstyle to merge things regarding github actions. I'll let him do the merge, especially because I don't yet fully comprehend where they merge stuff exactly at which step of the release process. (which they are fully in right now). 

@dragonstyle you can merge when you're done.

**Comment by dragonstyle** [comment]
Looks great!

**Comment by yihui** [comment]
@dragonstyle Thanks!

@cderv Could you update https://github.com/rstudio/tinytex/blob/main/.github/workflows/build.yaml accordingly when you have time? Thanks!

**Comment by cderv** [comment]
Sure !

**Pull request merged**
